Assets/_Functionalities/Wms/Scripts/WMSTileDataLayer.cs

```diff
@@ -14,6 +14,11 @@ public class WMSTileDataLayer : ImageProjectionLayer
     {
         private const string DefaultEpsgCoordinateSystem = "28992";
 
+        private Dictionary<string, string> customHeaders = new Dictionary<string, string>();
```

Comment:
Klopt het dat dit nog niet gebruikt wordt? En zijn de helper methodes nodig als je hier NameValueCollections van maakt en die in de config opbouw van Uxios doorgeeft?